src/Extensions/TimedScopes/Internal/TimedScopeEventSource.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics.Tracing;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.TimedScopes
+{
+	[EventSource(Name = "Microsoft-OMEX-TimedScopes")]
+	internal sealed class TimedScopeEventSource : EventSource
+	{
+		public static TimedScopeEventSource Instance { get; } = new TimedScopeEventSource();
+
+
+		[Event((int)EventSourcesEventIds.LogTimedScopeEventId, Level = EventLevel.Informational, Version = 3)]
+		public void WriteTimedScopeEvent(
+			string name,
+			string subType,
+			string metadata,
+			string userHash,
+			string serviceName,
+			string result,
+			string correlationId,
+			long durationMs) =>
```

Comment:
I think `durationInMilliseconds` would be clearer. Or better still, use a `TimeSpan`

Reply:
Since it's an event source parameter name is part of the contract and renaming it would be a breaking change, that I would like to avoid, and I think it does not support TimeSpan type